Source/WebCore/animation/TimelineRange.cpp

```diff
@@ -85,13 +86,17 @@ static const std::optional<CSSToLengthConversionData> cssToLengthConversionData(
 
 Length SingleTimelineRange::lengthForCSSValue(RefPtr<const CSSPrimitiveValue> value, RefPtr<Element> element)
 {
-    if (!value || value->isCalculated() || !element)
+    if (!value || !element)
         return { };
     if (value->valueID() == CSSValueAuto)
         return { };
 
+    element->document().updateLayoutIgnorePendingStylesheets();
```

Comment:
`protectedDocument()`

Reply:
What happened to moving this up to `SingleTimelineRange::parse()`?

Reply:
Fixed. Moving it farther up caused a test failure.